src/renderer/pages/project.vue

```
@@ -93,23 +93,21 @@
                       <!-- Undo/Redo buttons -->
                       <Button
                         variant="ghost"
-                        size="sm"
+                        size="lg"
                         :disabled="!mulmoScriptHistoryStore.undoable"
                         @click="mulmoScriptHistoryStore.undo"
                       >
                         <Undo
-                          :size="16"
                           :class="mulmoScriptHistoryStore.undoable ? 'text-foreground' : 'text-muted-foreground'"
                         />
```

ESLint

[error] Replace `⏎··························:class="mulmoScriptHistoryStore.undoable·?·'text-foreground'·:·'text-muted-foreground'"⏎·······················` with `·:class="mulmoScriptHistoryStore.undoable·?·'text-foreground'·:·'text-muted-foreground'"`

(prettier/prettier)